src/walk.ts

```diff
@@ -267,4 +261,78 @@ export function walkExportDeclaration(
   }
 
   setExport()
+
+  function resolveBindings(
```

Comment:
Let's have `extractIdentifiers` first.
You can refer to https://github.com/vuejs/core/blob/1f6a1102aa09960f76a9af2872ef01e7da8538e3/packages/compiler-core/src/babelUtils.ts#L208